### DISCUSSION TITLE
New user - can someone point me in the right direction? Issues with forms and a SQL database

### DISCUSSION THREAD

**Discussion by Sean-Roberts-89** [Budibase/budibase#9866] [Help]
Edit: I think I may have figured out the issue, so mods can delete this if you want.

I created bindings with a query and I had an option selected incorrectly on the options picker. 

Hello all, thanks for the help.

Right now I am having trouble with some basic functionality. 

I have a work_order table in a SQL database

work_order_id(PK)
work_order
amount_produced

I would like a user to be able to select a work_order from a options picker, and then modify it to add an amount produced. 

I'd then like to modify a few other tables, with more complexity but I'm just trying to understand this basic functionality before I go any further. 

I created an options picker which is nested in a worm and a data provider. 

I also have a number field.

I have a button which saves the row, the data source is the form, the table is work_order

column 1 is work_order_id which is set as the value {{ Work Order Select.Value }}
column 2 is amount_produced which is set to the value {{ Work Order Select.Fields.amount_produced }}

My issue is that 

1. This is inserting the work_order_id into the work_order section.
2. This endlessly creates new rows instead of updating the selected one. 

I've also seen some people use queries / bindings to accomplish CRUD tasks in the database, is this something I should explore?

**Comment by melohagan** [comment]
Hey @Sean-Roberts-89 

Would you mind providing some screenshots with captions showing what you did to get it working for you? It could be useful for other users to see!
